src/ui/public/notify/__tests__/notifier.js

```diff
@@ -33,10 +33,13 @@ describe('Notifier', function () {
 
   beforeEach(function () {
```

Comment:
same question as for the index pattern tests, how do these notifier tests relate to the agg changes?